lua/utils/security.lua

```
@@ -0,0 +1,207 @@
+  -- Redact hostnames
+  local hostnames = {
+    vim.fn.hostname()
+  }
+  
+  -- Add environment hostnames if env module is available
+  if env then
+    table.insert(hostnames, env.get("NVIM_HOSTNAME"))
+    table.insert(hostnames, env.get("NVIM_MAIN_LAPTOP"))
+    table.insert(hostnames, env.get("NVIM_STATIONARY"))
+  end
+  
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)